src/views/Feed/FeedContent.js

```diff
@@ -52,16 +52,13 @@ FeedContent.propTypes = {
 
   /** Shorthand for FeedExtra with prop images. */
   extraImages: customPropTypes.all([
-    customPropTypes.mutuallyExclusive(['children', 'content', 'extraText']),
-    PropTypes.oneOf([
-      PropTypes.array,
-      PropTypes.node,
-    ]),
+    customPropTypes.mutuallyExclusive(['children', 'content']),
```

Comment:
Removed exclusive with extraText.  Extra images and Extra text can appear together:

![image](https://cloud.githubusercontent.com/assets/5067638/17532795/15c12f86-5e37-11e6-8022-89b9708ced30.png)